tests/contracts/deployment.nim

```diff
@@ -0,0 +1,16 @@
+import std/os
+import pkg/ethers
+
+const hardhatMarketAddress = Address.init("0x59b670e9fA9D0A427751Af201D676719a970857b").get()
+const marketAddressEnvName = "CODEX_MARKET_ADDRESS"
+
+proc marketAddress*(): Address =
```

Comment:
This suggestion wouldn't change the functionality as it stands, but changing the signature might be a bit nicer if other contracts are added:

```nim
proc address*(_: type Marketplace): Address =
```
so that it would be called like:
```nim
# tests/contracts/testContracts.nim
marketplace = Marketplace.new(Marketplace.address, provider.getSigner())
```